src/PlanView/PlanView.qml

```diff
@@ -795,7 +795,7 @@ Item {
                         map:            editorMap
                         masterController:  _planMasterController
                         missionItem:    object
-                        width:          parent.width
+                        width:          parent ? parent.width : 0
```

Comment:
@holden-zenith if the parent is null then this component shouldn't exist either, rather than setting the width to 0 can you remove it?